tls/autoregenerate-after-expiry/autoregenerate-after-expiry.md

```diff
@@ -3,7 +3,7 @@
 ## Table of Contents
   - [How to meet the requirement](#How-to-meet-the-requirement)
   - [Items Do NOT Meet the Requirement (258)](#Items-Do-NOT-Meet-the-Requirement-258)
-    - [ (4)](#-4)
+    - [Unknown (4)](#Unknown-4)
```

Comment:
nit: how about `Unknown Owner` ? (at least for me that would read better)

Reply:
Good idea, updated